api.md

```diff
@@ -928,23 +928,30 @@ it is not possible to chain multiple commands together in one call.
 |Name|Description|
 |---|---|
 | `node_list` | An array of data nodes where the command should be executed. Defaults to all data nodes if not specified. |
+| `transactional` | Allows to specify if the execution of the statement should be transactional or not. Defaults to TRUE. |
 
 #### Sample Usage [](distributed_exec-examples)
 
 Create the role `davide` across all data nodes in a distributed database:
 
 ```sql
-SELECT distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
+CALL distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
 
 ```
 
 Create the role `davide` on two specific data nodes:
```

Comment:
Likely creating roles only on a limited set of data nodes will require to set grants specific to them as well. I am wondering if it needs to be covered somewhere

Reply:
@pmwkaa I think your comment is more relevant to https://github.com/timescale/docs.timescale.com-content/pull/562